### DISCUSSION TITLE
ROWY: How to reference - map - Data types in Flutterflow

### DISCUSSION THREAD

**Discussion by DayneLinx** [AntlerVC/firetable#1035] [Support / Q&A] [answered]
I need some guidance on how to map the headerImage (map) Datatype to FLutterflow so can reference the images array for display in an image gallery as well as a single image from the array.

I have so far:
1 Created a custom Data Type with the subfields shown
2 Added headerImage as an array type

I have not had success but i bet I am missing something really basic!

I have included screenshots but if you require additional shots and explanations I am ready! 

Thanks in advance
![headerImage - Map Data Type](https://user-images.githubusercontent.com/23277612/209131248-8ae71b2a-f7b9-44b3-b6ba-47214ffe1dba.png)
![hearImage - Arrray](https://user-images.githubusercontent.com/23277612/209131253-713f3d32-cf24-4f8e-9fc3-06496ae01915.png)
![Screenshot 2022-12-19 at 18 39 16](https://user-images.githubusercontent.com/23277612/209131254-87467e01-e0d8-4575-9ca0-583eb1e8bb52.png)
![Screenshot 2022-12-19 at 18 39 27](https://user-images.githubusercontent.com/23277612/209131256-4a08dad4-7f74-4a76-ac5f-4744bc80d262.png)

**Comment by BeeBombshell** [comment]
Hey @DayneLinx, we are currently working on documentation specific for Rowy x FlutterFlow users. This would also include a mapping for FlutterFlow data types in Rowy. Hope it'd be helpful. Would let you know once its out!

**Comment by DayneLinx** [comment]
HI, Bhavya

That documentation will be much appreciated! Please, keep me in the loop for when it is available. Will this be expected at Q1 2023.

Thanks

From: Bhavya Verma ***@***.***>
Date: Wednesday, 4 January 2023 at 10:09
To: rowyio/rowy ***@***.***>
Cc: DayneLinx ***@***.***>, Mention ***@***.***>
Subject: Re: [rowyio/rowy] ROWY: How to reference - map - Data types in Flutterflow (Discussion #1035)

Hey @DayneLinx<https://github.com/DayneLinx>, we are currently working on documentation specific for Rowy x FlutterFlow users. This would also include a mapping for FlutterFlow data types in Rowy. Hope it'd be helpful. Would let you know once its out!

**Comment by BeeBombshell** [comment, marked as answer]
Hey @DayneLinx, we have recently added some documentation specifically for Rowy X FlutterFlow users. Here's the link to the data types mapping docs: https://docs.rowy.io/integrations/flutterflow/mapping-datatypes. Let us know what else would you like to be included in the docs, which would be helpful for the FlutterFlow X Rowy enthusiasts!

**Comment by DayneLinx** [comment]
Hi Bhavya

Thank you for this information, I will check it out.

Do you have any idea what this could be in your experience? https://github.com/FlutterFlow/flutterflow-issues/issues/608

Also, do you have anybody that could offer remote support for 30 mins or less this week?

Keep up the amazing work!

Kind Regards

--
CONFIDENTIAL: This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error, please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.

From: Bhavya Verma ***@***.***>
Date: Tuesday, 28 February 2023 at 08:55
To: rowyio/rowy ***@***.***>
Cc: DayneLinx ***@***.***>, Mention ***@***.***>
Subject: Re: [rowyio/rowy] ROWY: How to reference - map - Data types in Flutterflow (Discussion #1035)

Hey @DayneLinx<https://github.com/DayneLinx>, we have recently added some documentation specifically for Rowy X FlutterFlow users. Here's the link to the data types mapping docs: https://docs.rowy.io/integrations/flutterflow/mapping-datatypes. Let us know what else would you like to be included in the docs, which would be helpful for the FlutterFlow X Rowy enthusiasts!

**Reply by BeeBombshell** [reply]
Hey @DayneLinx,

> Do you have any idea what this could be in your experience? [FlutterFlow/flutterflow-issues#608](https://github.com/FlutterFlow/flutterflow-issues/issues/608)

Not very sure about this, since I do not have much experience using FlutterFlow. However, I can connect you with some of the Rowy X FlutterFlow Experts who provide paid services. Can you share your email so that I can share the list with you?

Also, we recently added a Rowy X FlutterFlow tutorial on YouTube: https://www.youtube.com/watch?v=CQuhFFBxGY0. Check it out, and do let us know if you find it helpful! 😊

**Comment by DayneLinx** [comment]
Hi Bhavya

Thank you for contacting me. I appreciate it. Today I logged in to Rowy and my data is gone! Please see the attached video for context.
https://www.icloud.com/iclouddrive/0bdaN0VMrUxJbIqy4arqViWSQ#Screen_Recording_2023-03-20_at_20.42
I have reviewed the video and like it, most of it I know already but the quality of the narration and tutoring is good.

Kind Regards

--
CONFIDENTIAL: This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error, please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.

From: Bhavya Verma ***@***.***>
Date: Monday, 20 March 2023 at 19:21
To: rowyio/rowy ***@***.***>
Cc: DayneLinx ***@***.***>, Mention ***@***.***>
Subject: Re: [rowyio/rowy] ROWY: How to reference - map - Data types in Flutterflow (Discussion #1035)

Hey @DayneLinx<https://github.com/DayneLinx>,

Do you have any idea what this could be in your experience? FlutterFlow/flutterflow-issues#608<https://github.com/FlutterFlow/flutterflow-issues/issues/608>

Not very sure about this, since I do not have much experience using FlutterFlow. However, I can connect you with some of the Rowy X FlutterFlow Experts who provide paid services. Can you share your email so that I can share the list with you?

Also, we recently added a Rowy X FlutterFlow tutorial on YouTube: https://www.youtube.com/watch?v=CQuhFFBxGY0. Check it out, and do let us know if you find it helpful! 😊

**Comment by DayneLinx** [comment]
HI Bhavya

I will disconnect and reconnect Rowy to see if this makes a difference.

--
CONFIDENTIAL: This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error, please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.

From: Bhavya Verma ***@***.***>
Date: Monday, 20 March 2023 at 19:21
To: rowyio/rowy ***@***.***>
Cc: DayneLinx ***@***.***>, Mention ***@***.***>
Subject: Re: [rowyio/rowy] ROWY: How to reference - map - Data types in Flutterflow (Discussion #1035)

Hey @DayneLinx<https://github.com/DayneLinx>,

Do you have any idea what this could be in your experience? FlutterFlow/flutterflow-issues#608<https://github.com/FlutterFlow/flutterflow-issues/issues/608>

Not very sure about this, since I do not have much experience using FlutterFlow. However, I can connect you with some of the Rowy X FlutterFlow Experts who provide paid services. Can you share your email so that I can share the list with you?

Also, we recently added a Rowy X FlutterFlow tutorial on YouTube: https://www.youtube.com/watch?v=CQuhFFBxGY0. Check it out, and do let us know if you find it helpful! 😊

**Comment by DayneLinx** [comment]
Hi Bhavya

I had to reconnect and relink to firebase and also redefine the fields also, any idea why that would suddenly occur? I also have an additional field added called ‘Test Field’ that I never created.

Kind Regards

--
CONFIDENTIAL: This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error, please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.

From: Bhavya Verma ***@***.***>
Date: Monday, 20 March 2023 at 19:21
To: rowyio/rowy ***@***.***>
Cc: DayneLinx ***@***.***>, Mention ***@***.***>
Subject: Re: [rowyio/rowy] ROWY: How to reference - map - Data types in Flutterflow (Discussion #1035)

Hey @DayneLinx<https://github.com/DayneLinx>,

Do you have any idea what this could be in your experience? FlutterFlow/flutterflow-issues#608<https://github.com/FlutterFlow/flutterflow-issues/issues/608>

Not very sure about this, since I do not have much experience using FlutterFlow. However, I can connect you with some of the Rowy X FlutterFlow Experts who provide paid services. Can you share your email so that I can share the list with you?

Also, we recently added a Rowy X FlutterFlow tutorial on YouTube: https://www.youtube.com/watch?v=CQuhFFBxGY0. Check it out, and do let us know if you find it helpful! 😊
